internal/utils/version.go

```diff
@@ -1,4 +1,4 @@
 package utils
 
 // Version of the hldbx tool
-const Version = "0.0.6"
+const Version = "1.0.0"
```

Comment:
```suggestion
const Version = "0.0.7"
```
Just heard today that we want to get artifact signing in before we bump to 1.0.0